app/views/partials/_properties.html.erb

```diff
@@ -5,6 +5,8 @@
       controller:  "<%= controller_name %>"
     },
     text: {
+      default_option: "<%= t('default_text.option') %>",
+      default_option_hint: "<%= t('default_text.option_hint') %>",
```

Comment:
I think we have already one of them available `MetadataPresenter::DefaultText[:hint]`.
Maybe we should put the other value in the gem? @brenetic do you agree?

Reply:
not sure i'm following why this was an issue?

![Screenshot 2021-04-01 at 09 44 39](https://user-images.githubusercontent.com/3466862/113268074-ea7e5c00-92ce-11eb-8677-750be7f27cc4.png)

it's ok for the text to be the same as the editor is going to change them or remove them

Reply:
If we can alter the template that outputs the radio and checkbox items so that we have something like this:

```
<div class="govuk-checkboxes" data-module="govuk-checkboxes">
    <div class="govuk-checkboxes__item EditableComponentCollectionItem">
        <input id="answers-checkbox-checkboxes-1-option-2-field" class="govuk-checkboxes__input" aria-describedby="answers-checkbox-checkboxes-1-option-2-hint" type="checkbox" value="Option 2" name="answers[checkbox_checkboxes_1][]" disabled="disabled">
        <label for="answers-checkbox-checkboxes-1-option-2-field" class="govuk-label govuk-checkboxes__label EditableElement" contenteditable="true" data-fb-default-text="[Required option text]">Option 2</label>
        <span class="govuk-hint govuk-checkboxes__hint EditableElement" id="answers-checkbox-checkboxes-1-option-2-hint" contenteditable="true" data-fb-default-text="[Optional hint text]">[Optional hint text]</span>
    </div>
    <div class="govuk-checkboxes__item EditableComponentCollectionItem">
        <input id="answers-checkbox-checkboxes-1-option-field" class="govuk-checkboxes__input" aria-describedby="answers-checkbox-checkboxes-1-option-hint" type="checkbox" value="Option" name="answers[checkbox_checkboxes_1][]" disabled="disabled">
        <label for="answers-checkbox-checkboxes-1-option-field" class="govuk-label govuk-checkboxes__label EditableElement" contenteditable="true" data-fb-default-text="[Required option text]">Option</label>
        <span class="govuk-hint govuk-checkboxes__hint EditableElement" id="answers-checkbox-checkboxes-1-option-hint" contenteditable="true" data-fb-default-text="[Optional hint text]">[Optional hint text]</span>
    </div>
</div>

```

Then I should be able change the JS to pick up from those element. Not sure on effort as untested though.
(see required data attributes on label and span)
```
data-fb-default-text="[Required option text]"
data-fb-default-text="[Optional hint text]"
```

Alternative is to add these attributes (with different names, obviously) to the outer component, but I think that might be harder and is still deviating from the method other elements are using, so we might not be gaining much.

E.g.

```
<div class="govuk-checkboxes" data-module="govuk-checkboxes" data-fb-default-label-text="[Required option text]" data-fb-default-hint-text="[Optional hint text]">
    <div class="govuk-checkboxes__item EditableComponentCollectionItem">
    ...
    </div>
</div>
```

Reply:
Note: Lines 73 + 74 of app/javascript/src/page_editable_content.js can be adapted to find values from data-fb-default-text attributes of the <label> and <span> elements, if we figure out how to add them through the fb-metadata-presenter/app/views/metadata_presenter/component/_radios.html.erb (and _checkboxes.html.erb) templates.